languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/multitoken-suggest.txt

```diff
@@ -129,7 +130,7 @@ Kleene'schen Hüllen
 New Yorkerin
 New Yorkerinnen
 San Martín
-Schimon Stein
+Shimon Stein
```

Comment:
In de.wikipedia.org, Schimon Stein. In en.wikipedia.org, Shimon Stein. Different transcriptions? Or a misspelling?

Reply:
Now I noticed that you keep both.